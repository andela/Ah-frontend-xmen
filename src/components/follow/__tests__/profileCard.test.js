import { shallow } from 'enzyme';
import React from 'react';
import ProfileCard from '../profileCard';

describe('Profile Card', () => {
  const profile = {
    username: 'khaleesi',
    first_name: 'daenerys',
    last_name: 'stormborne',
    is_following: false,
  };
  const props = {
    profile,
    onFollow: jest.fn(),
    onUnfollow: jest.fn(),
  };

  it('renders without crashing', () => {
    const wrapper = shallow(<ProfileCard {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
