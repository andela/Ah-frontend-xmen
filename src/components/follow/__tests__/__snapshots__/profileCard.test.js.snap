// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Card renders without crashing 1`] = `ShallowWrapper {}`;
